.idea/deployment.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Create `.gitignore` and add this folder `.idea`. Commit and push. Then remove it from GitHub.
Or the same and remove the folder from local folder, then commit and push.